pop-api/src/lib.rs

```diff
@@ -32,6 +33,7 @@ impl ink::env::chain_extension::FromStatusCode for PopApiError {
 	fn from_status_code(status_code: u32) -> core::result::Result<(), Self> {
 		match status_code {
 			0 => Ok(()),
+			5 => Err(PopApiError::CallFiltered),
```

Comment:
Where does this value originate from? Assume it is related to some call filtered error via dispatch, but might need to ensure that this won't change if being relied upon from subsystem.

Reply:
Originates from frame_system. Added comment and updated name.